sql/hive-thriftserver/v1.2.1/src/main/java/org/apache/hive/service/cli/ColumnValue.java

```diff
@@ -202,6 +202,8 @@ public static TColumnValue toTColumnValue(Type type, Object value) {
     case UNION_TYPE:
     case USER_DEFINED_TYPE:
       return stringValue((String)value);
+    case NULL_TYPE:
+      return stringValue((String)value);
```

Comment:
Hive 2.3 has fixed this issue:
https://github.com/apache/spark/blob/3f102a82295a8b775d00d86683561a71303f3e39/sql/hive-thriftserver/v2.3.5/src/main/java/org/apache/hive/service/cli/ColumnValue.java#L209-L210